src/services/apiService.ts

```diff
@@ -20,33 +24,82 @@ export class ApiService {
     }
 
     public postTransaction(transaction: Transaction): Promise<void> {
-        return new Promise(async (resolve, reject) => {
+        return new Promise(async (resolved, rejected) => {
```

Comment:
Why are these parameter names changed?

```suggestion
        return new Promise(async (resolve, reject) => {
```

Reply:
Params were changed to comply with ts lint errors.